docs/docs/genai/eval-monitor/scorers/llm-judge/alignment.mdx

```diff
@@ -168,6 +168,10 @@ traces_for_alignment = mlflow.search_traces(
 # Align the judge using human corrections (minimum 10 traces required)
 if len(traces_for_alignment) >= 10:
     optimizer = SIMBAAlignmentOptimizer(model="anthropic:/claude-opus-4-1-20250805")
+
+    # Run alignment - shows minimal progress by default:
+    # INFO: Starting SIMBA optimization with 15 examples (set logging to DEBUG for detailed output)
+    # INFO: SIMBA optimization completed
```

Comment:
Question (maybe not related to this PR): Do we show anywhere in the documentation that there is a default optimizer? I.e., that the user can call:

```
judge.align(traces)
```
?

Reply:
I don't think that we mention that. I'll file a PR for that. Good call-out!

Reply:
#17814 for the follow-up!